src/Adapter/MSTest.TestAdapter/Execution/TestMethodInfo.cs

```diff
@@ -412,7 +412,9 @@ private static TestFailedException HandleMethodException(Exception ex, Exception
             && RuntimeContext.IsHotReloadEnabled
             && missingMethodException.StackTrace?.IndexOf(Environment.NewLine, StringComparison.Ordinal) is { } lineReturnIndex
             && lineReturnIndex >= 0
+#pragma warning disable IDE0057 // Use range operator
```

Comment:
@SimonCropp I am not sure why the test adapter project (although polyfill is available transitively) is not picking up range and indexing correctly.